extensions/amp-youtube/1.0/test/test-youtube.js

```diff
@@ -155,4 +157,41 @@ describes.realWin('YouTube preact component v1.0', {}, (env) => {
       event: 'listening',
     });
   });
+
+  it('exposes properties from info messages', () => {
+    const ref = createRef();
+
+    const wrapper = mount(
+      <Youtube
+        ref={ref}
+        videoid="IAvf-rkzNck"
+        shortcode="B8QaZW4AQY_"
+        style={{width: 500, height: 600}}
+      />,
+      {attachTo: document.body}
+    );
+
+    const {contentWindow} = wrapper.getDOMNode().querySelector('iframe');
+
+    expect(ref.current.currentTime).to.equal(0);
+    expect(ref.current.duration).to.be.NaN;
+
+    mockMessage(window, contentWindow, {info: {duration: 420}});
+
+    expect(ref.current.currentTime).to.equal(0);
+    expect(ref.current.duration).to.equal(420);
+
+    mockMessage(window, contentWindow, {info: {currentTime: 12.3}});
+
+    expect(ref.current.currentTime).to.equal(12.3);
+    expect(ref.current.duration).to.equal(420);
+  });
 });
+
+function mockMessage(win, source, data) {
```

Comment:
General question, doesn't need to be addressed in this PR: is there a standard shared test utils for Bento components where something like this might live/be found?

Reply:
I think it's more generally shareable. I've added an item to track.